### PR TITLE
Update FxCop analyzers VSIX (Microsoft Code Analysis VSIX) to work on…

### LINKED ISSUE #2034: Vsix doesn't support VS 2019
I'm getting this:

image

### PR DESCRIPTION
**Pull request by mavasani**
… VS2019

Fixes #2034

TODOs:
1. ~~Create a new fwlink for the 2019 VSIX marketplace link and update the manifest~~ Using the docs link now.
2. Queue a new signed build with BUILDING_VSIX envinronment variable set to 'true', so we only enable a small subset of analyzers by default (matching the ones in 2017 VSIX)

### CODE REVIEW

File: src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/Microsoft.CodeAnalysis.FxCopAnalyzers.Setup.csproj [hunk 1]
```diff
@@ -29,7 +29,7 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudio.Shell.15.0" Version="15.0.26201" />
+    <PackageReference Include="Microsoft.VisualStudio.Shell.15.0" Version="16.0.28729" />
```
Comment (sharwell) [inline src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/Microsoft.CodeAnalysis.FxCopAnalyzers.Setup.csproj]: ❓ Why not shell 16.0?

Comment (mavasani) [reply src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/Microsoft.CodeAnalysis.FxCopAnalyzers.Setup.csproj]: I didn't find one on NuGet.org, and didn't find it even in my VS2019 install. Probably the number was never bumped up?

File: src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/source.extension.vsixmanifest [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
     <Identity Id="4db2d63d-3320-4fbd-bf80-07f8d1500bd3" Version="|%CurrentProject%;GetVsixVersion|" Language="en-US" Publisher="Microsoft"  />
-    <DisplayName>Microsoft Code Analysis 2017</DisplayName>
+    <DisplayName>Microsoft Code Analysis 2019</DisplayName>
     <Description xml:space="preserve">Live code analysis rules and code fixes addressing API design, performance, security, and best practices for C# and Visual Basic.</Description>
     <MoreInfo>https://go.microsoft.com/fwlink/?linkid=849061</MoreInfo>
```
Comment (mavasani) [inline src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/source.extension.vsixmanifest]: Tagging @gewarren - would you be able to help in creating a new fwlink for the "Microsoft Code Analysis 2019" VSIX? Current one points to VS2017 release. We also need to eventually update the documentation at https://docs.microsoft.com/en-us/visualstudio/code-quality/install-fxcop-analyzers?view=vs-2019#to-install-fxcop-analyzers-as-a-vsix. I can file a new tracking docs issue for the same.

Comment (mavasani) [reply src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/source.extension.vsixmanifest]: I am actually going to replace the fwlink with https://docs.microsoft.com/en-us/visualstudio/code-quality/install-fxcop-analyzers?view=vs-2019#to-install-fxcop-analyzers-as-a-vsix so we don't need to produce such an fwlink with every release. The documentation would need to be updated with each release to point to all available VSIX releases.

Comment (gewarren) [reply src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/source.extension.vsixmanifest]: Yes I will update that page for VS 2019. Thanks for letting me know.

**Comment by mavasani** [comment]
Ping for reviews. I'd like to get this in to start driving the VS2019 VSIX testing.

**Comment by mavasani** [comment]
Thanks!

**Comment by sinand99** [comment]
One month since VS2019 RTM and Microsoft still does not bother to update one of the most important extensions. Please prioritize this.

**Pull request merged**